src/collection-preferences/content-display/__integ__/pages/content-display-page.ts

```diff
@@ -15,7 +15,7 @@ export default class ContentDisplayPageObject extends CollectionPreferencesPageO
     );
     // Using getHTML because getText returns an empty string if the live region is outside the viewport.
     // See https://webdriver.io/docs/api/element/getText/
-    return expect(liveRegion.getHTML(false)).resolves.toBe(announcement);
+    return expect(liveRegion.getHTML()).resolves.toContain(announcement);
```

Comment:
Why this change?

Reply:
The getHTML() api changes. In v7 it took a flag and in v9 it takes an object.

When the flag is provided it returns the inner part of the node only. When not provided - the entire node. That's why I changed toBe to toContain. We can change this back when the webdriver dependency is updated, although that is not really necessary.